libclock_gettime_wrapper/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: libclock_gettime_wrapper
+  version: 2.17
+
+source:
+  - path: ./
+
+build:
+  number: 1
+  string: {{ PKG_BUILDNUM }}
+  
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+about:
+  home: open-ce
+  license: apache-2
+  summary: summary
+  description: |
+    This is a wrapper package to map clock_gettime_GLIBC_PRIVATE to GLIBC_2.17
+  doc_url: 
```

Comment:
Do we want to point to some reference doc for this?

Reply:
yes, that needs filled in